TestObjectiveDropbox/IntegrationTests/TestClasses.m

```diff
@@ -1974,7 +1995,11 @@ - (void)groupsDelete:(void (^)(void))nextTest {
             if (result) {
                 MyLog(@"%@\n", result);
                 if ([result isInProgress]) {
-                    [TestFormat abort:error routeError:routeError];
+                    if (tries >= 3) {
```

Comment:
this was added because the delete may not have completed yet and we should retry fetching the async task status